app/internal/page/restrictions.go

```diff
@@ -24,6 +25,7 @@ func Restrictions(tmpl template.Template, lpaStore LpaStore) Handler {
 			App:          appData,
 			Restrictions: lpa.Restrictions,
```

Comment:
would it be worth removing this field and updating the template to use `Lpa.Restrictions` so you don't repeat yourself in tests?

Reply:
Yup good point, will update 👍